### DISCUSSION TITLE
Unable to display images in body of the release API response

### DISCUSSION THREAD

**Discussion by horlah** [community/community#108336] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I'm working on a prototype that enables people to manage Github releases. While trying to read releases from Github, in situations where there is an image in the release note, when I get a release from the API, I'm unable to render the image. When I load the image on the browser directly, the URL gets updated, more like it goes through an authentication when loads the image.

This is a release body response for a release for example:
```
![colors_loader](https://github.com/horlah/docus-test/assets/17768868/05d9d1dd-898b-494b-a41b-38b0370b0622)

Seeing how image will work.

**Full Changelog**: https://github.com/horlah/docus-test/commits/v0.01
```

The image in this markdown only loads on Github, but doesn't load elsewhere, e.g when trying to render in html, outside of Github
```
<img src="https://github.com/horlah/docus-test/assets/17768868/05d9d1dd-898b-494b-a41b-38b0370b0622" alt="test image" />
```

**Comment by spenserblack** [comment]
Right now it looks like https://github.com/horlah/docus-test is a private repository, so anyone would need authorization to view that repository and its contents (like the images in `/assets`). Do you still have issues if you reference a public image?

**Reply by horlah** [reply]
No I don't, thanks for the update!

But is there there a way to work around this?
If I want an un-authenticated user to access this release images, for the purpose of making release messages alone public? @spenserblack

**Reply by spenserblack** [reply, marked as answer]
> I want an un-authenticated user to access this release images, for the purpose of making release messages alone public?

I think that, from the user's perspective, they *would* want to be authenticated to be able to view private images. If there was a workaround (and I don't think there is), that would probably be a pretty big problem at GitHub where unauthenticated users could view images that are supposed to be private. The users can implement their own workarounds that suit their needs, like making sure that any images that should be visible publicly are not in private repositories.

**Reply by horlah** [reply]
Thank you so much for the clarity @spenserblack
